### ISSUE TITLE
[CAIA A11y Research - PILOT]: Benefits Condition Team: Conditions Information Entry (Mobile) (November 2024)

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#97582]
> [!IMPORTANT]
> - **TICKET STATUS:** `COMPLETED`
> - **PILOT DATE:**  November 25, 12-1 PM
> - **TYPES OF ASSISTIVE TECH:** Info on AT being tested in user sessions needed, to prep and conduct pilot.
> - **FINDINGS & FEEDBACK:** [See comment in this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/97582#issuecomment-2498765176)

# CAIA A11y Support
This is a child ticket of the research ticket [96633](https://github.com/department-of-veterans-affairs/va.gov-team/issues/96633) - created to track work, completed from this pilot session.
> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- Pilot: @mutiaadhocteam 
- Co-pilot: @sara-amanda 

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name:Condition Team
- OCTO-DE product owner:Jen Bertsch
- Product manager: Yurena Garcia
- Designer:Sudeepti Bhatnagar
- FE engineer:William Phelps
- Product/team Slack channel: https://dsva.slack.com/archives/C04AZ8T7XN1

...

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:** November 25
- **Prototype**: [Staging](https://staging.va.gov/user-testing/new-conditions )
- [**Research Convo Guide**](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/research/CC-Team-Research/2024-11%20Conditions%20Info%20Entry/2024-11%20Conversation%20Guide%20-%20Conditions%20info%20entry.md)
-  [**Research Plan**](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/research/CC-Team-Research/2024-11%20Conditions%20Info%20Entry/2024-11%20Research%20Plan%20-%20Condition%20info%20entry.md) 

</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y:**
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] **Determine date and time for pilot:**
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [x] **Review research** [conversation guide](link needed) and provide feedback in comments.
- [x] [Request AT device and tech to be used in sessions](LINK)
- [x] [Receive AT device and tech from Team in research ticket](LINK) to be used in sessions to prep for pilot
- [x] **Review prototype** in `Location` - `STATUS OF PROTOTYPE`
- [x] **Run pilot** with VFS Researcher
- [x] **Optional: [Note Taking](https://docs.google.com/document/d/11a9r_w6Vd8udBMmnz86RITSXHP37amGcX0FU8RLqHgE/edit?usp=sharing)**
```

**Comment by sara-amanda** [comment]
## Pilot Takeaways

### Radio Buttons read as checkboxes
- **Finding**: 
    - This implies multi-selectability. 
    - This does not accurately reflect the functionality of the component.
- **Recommendation**: Update from checkbox to radio buttons. 
- **Next Step**: VADS Ticket - Radio buttons, being called a checkbox (which implies multiple) when in actuality it is a radio button (which would imply). 

### Star required (*required) is not connected to its according text
- Therefore, it is not announced alongside the according text.

### Verbiage: Selected or Checked
- **Finding**: 
    - Instead of saying selected or unselected it says checked or unchecked 
    - Users will have difficulty understanding that only one option can be selected.
    - **Example**: Which side of the body is your condition on?
- **Recommendation**: Change verbiage to selected or unselected.

### Focus Placement: 
- **Finding**: 
    - Focus is sent to “Review” instead of “Review your conditions.” 
    - Modal: Focus first going to the closed button, instead of the Modal heading
- **Recommendation**: Focus should be placed on the heading of the Modal. 

### Heading Order: 
- **Finding**: 
    - Focus jumps to the bottom of the page, ignoring the content you are needing to review in the middle of the page. 
    - Users of AT in iOS for example, use the rotor and headings to skim the page. 
    - **Example**: It is going from you can claim a condition isn’t listed to need help. 
- **Recommendation**: Manage focus to semantically go down the page. 

### Button Placement
- **Finding**: 
    - The back button is right next to the continue button. 
    - For sighted users this works great. 
    - As a non-sighted user, they may have to click through all the components to get to this point. Newer screen-reader users may not know how to get back. 
- **Recommendation**: The back button should be located near the top of the screen. 
- **Next Steps**: VADS ticket

### Error Trigger and Placement
- Finding: 
    - When clicking through the checkboxes on the cause screen, it triggered an error on the components themselves.
    - Focus is on the first selection item: “My condition was caused by,” and when I go to the next checkbox, it triggers the error, that you must enter a response, which can be disruptive to users. 
- **Recommendation**: Place the error to trigger on the continue button. 

### Verbose Character Count
- **Finding**: 
    - As the user is typing, it should not be announced how many characters are remaining. 
- **Recommendation**: 
    - Remove announcement, and announce only when the user tabs or moves out of the text box. 
    - Make the number of characters an item that can have focus and inform the user, upon their request. 
- **Next Steps**: Recommendation to VADS via ticket.

### iOS Specific Issue - Focus Jumps
- **Example**: Date picker: January 2024 it jumps to need help. 
- **Next Steps**: Known issue

### Bulleted Lists
- **Finding**: Bulleted lists are reading as black square. 
- **Recommendation**: Update them to read as bullet points in the list. 

### Multiple Components in (Cherry Version) 
- **Finding**: 
    - Multiple components on one page, can be confusing, so the first demo was better. 
    - Appearing on the same page because it is not broken up, and it would be confusing. 
    - Hard to know things below are changing 
    - Example: The drop down with option one vs no drop down after selection two. 
- **Recommendation**: Avoid conditional content on the page and keep it simple with one thing per page.
